### DISCUSSION TITLE
ArgoCD ubuntu base image CVEs

### DISCUSSION THREAD

**Discussion by dmarquez-splunk** [argoproj/argo-cd#17285] [General]
Hi,

We are currently receiving vulnerability reports from your internal prodsec team for vulnerabilities in ArgoCD's ubuntu base images. Are there any plans in the near future to upgrade the base image to the latest ubunut OS version? Below are some of the CVEs reported; I look forward to hearing any ideas on how we can best address these moving forward!

https://ubuntu.com/security/CVE-2018-1000021
https://ubuntu.com/security/CVE-2018-6952
https://ubuntu.com/security/CVE-2021-45261
https://ubuntu.com/security/CVE-2022-3219
https://ubuntu.com/security/CVE-2023-29383
https://ubuntu.com/security/CVE-2023-51767
https://ubuntu.com/security/CVE-2023-7008

**Comment by crenshaw-dev** [comment]
What Argo CD version are you running?

**Reply by dmarquez-splunk** [reply]
This was scanned on 2.10.0

**Reply by manish222agr** [reply]
We have vulnerabilities for 2.7.17 as well.

**Comment by dmarquez-splunk** [comment]
@crenshaw-dev any updates on this that I can relay to our prodsec team?
